app/models/internal/event.rb

```diff
@@ -141,6 +137,17 @@ def end_after_start
       end
     end
 
+    def dates_in_future
```

Comment:
I don't feel strongly about it but using something like the [validates_timeliness](https://github.com/adzap/validates_timeliness) gem would make this cleaner